lib/trufi_app.dart

```diff
@@ -98,7 +99,9 @@ class TrufiApp extends StatelessWidget {
         BlocProvider<RequestManagerCubit>(
           create: (context) => RequestManagerCubit(
             OfflineRepository(),
-            OnlineRepository(),
+            (TrufiConfiguration().generalConfiguration.typeServer == TypeServer.defaultServer)
```

Comment:
Do we need the braces around that statement?
```suggestion
            TrufiConfiguration().generalConfiguration.typeServer == TypeServer.defaultServer
```